addons/auth_signup/models/res_users.py

```diff
@@ -281,62 +282,16 @@ def send_unregistered_user_reminder(self, after_days=5, batch_size=100):
     def _alert_new_device(self):
         self.ensure_one()
         if self.email:
-            email_values = {
-                'email_cc': False,
-                'auto_delete': True,
-                'message_type': 'user_notification',
-                'recipient_ids': [],
-                'partner_ids': [],
-                'scheduled_date': False,
-                'email_to': self.email
-            }
+            mail = self._notify_security_setting_update(
+                subject=_('New Connection to your Account'),
+                content=_('A new device was used to sign in to your account.'),
+                mail_values={'scheduled_date': fields.Datetime.now() + timedelta(minutes=1)},
+            )
```

Comment:
Can't we also force_send here if totp is disabled? This way we cover all cases where you're functionally logged in.

base http already checks that indirectly via `_mfa_url`